source/whats-new.txt

```diff
@@ -28,7 +28,7 @@ New features of the 4.3 Java driver release include:
 - Added a ``setWindowFields`` pipeline stage to allow the use of window operators 
 - Added support for setting Netty `io.netty.handler.ssl.SslContext <https://netty.io/4.1/api/io/netty/handler/ssl/SslContext.html>`__
 - Added ``snapshot`` (non-speculative) as a ``readConcern`` level for read commands outside of transactions
```

Comment:
This isn't completely accurate: that read concern level was actually added a long time ago.  The new feature is (indirect) support for `atClusterTime` for the read concern, which is supported via a new ClientSessionOption.